portal-api/api-tests/v1/gef/application.api-test.js

```diff
@@ -0,0 +1,271 @@
+  //       const titleUpdate = {
+  //         title: 'Confirm eligiblity (mandatory criteria)',
+  //       };
+
+  //       await as(anEditor).post(mandatoryCriteria).to(baseUrl);
+  //       await as(anEditor).put(titleUpdate).to(`${baseUrl}/${mandatoryCriteria.id}`);
+
+  //       const { status, body } = await as(anEditor).get(`${baseUrl}/${mandatoryCriteria.id}`);
+
+  //       expect(status).toEqual(200);
+  //       expect(body).toEqual(expectMongoId({
+  //         ...mandatoryCriteria,
+  //         ...titleUpdate,
+  //       }));
+  //     });
+
+  //     it('works for items', async () => {
+  //       const mandatoryCriteria = allMandatoryCriteria[2];
+  //       const itemUpdate = {
+  //         version: 99,
+  //         dateCreated: "2021-01-01T00:00",
+  //         isInDraft: true,
+  //         title: 'test 99',
+  //         htmlText: `<p>Test is a mock test</p>`
+  //       };
+
+  //       await as(anEditor).post(mandatoryCriteria).to('/v1/gef/mandatory-criteria-versioned');
+  //       await as(anEditor).put(itemUpdate).to(`/v1/gef/mandatory-criteria-versioned/${mandatoryCriteria.id}`);
+
+  //       const { status, body } = await as(anEditor).get(`/v1/gef/mandatory-criteria-versioned/${mandatoryCriteria.id}`);
+
+  //       expect(status).toEqual(200);
+  //       expect(body).toEqual(expectMongoId({
+  //         ...mandatoryCriteria,
+  //         ...itemUpdate,
+  //       }));
+  //     });
+
+  //   })
+  // });
+
+  // describe('DELETE /v1/gef/mandatory-criteria-versioned/:id', () => {
+  //   it('rejects requests that do not present a valid Authorization token', async () => {
+  //     const { status } = await as().remove('/v1/gef/mandatory-criteria-versioned/1');
+
+  //     expect(status).toEqual(401);
+  //   });
+
+  //   it('rejects requests that present a valid Authorization token but do not have "editor" role', async () => {
+  //     await as(anEditor).post(newMandatoryCriteria).to('/v1/gef/mandatory-criteria-versioned');
+
+  //     const { status } = await as(noRoles).remove('/v1/gef/mandatory-criteria-versioned/1');
+
+  //     expect(status).toEqual(401);
+  //   });
+
+  //   it('accepts requests that present a valid Authorization token with "editor" role', async () => {
+  //     await as(anEditor).post(newMandatoryCriteria).to('/v1/gef/mandatory-criteria-versioned');
+
+  //     const { status } = await as(anEditor).remove('/v1/gef/mandatory-criteria-versioned/1');
+
+  //     expect(status).toEqual(200);
+  //   });
+
+  //   it('deletes the mandatory-criteria', async () => {
+  //     await as(anEditor).post(newMandatoryCriteria).to('/v1/gef/mandatory-criteria-versioned');
+  //     await as(anEditor).remove('/v1/gef/mandatory-criteria-versioned/1');
+
+  //     const { status, body } = await as(anEditor).get('/v1/gef/mandatory-criteria-versioned/1');
+
+  //     expect(status).toEqual(200);
+  //     expect(body).toEqual({});
+  //   });
+  // });
+});
```

Comment:
should this all be commented out?

Reply:
For me to work on the FE I had to merge matts work into mine, so make sure you leave this comment on his PR, unless of course he has already resolved this issue.